client/transport/src/ws/mod.rs

```diff
@@ -195,6 +196,16 @@ impl TransportSenderT for Sender {
 		Ok(())
 	}
 
+	/// Sends out a ping request. Returns a `Future` that finishes when the request has been
+	/// successfully sent.
+	async fn send_ping(&mut self, data: &[u8]) -> Result<(), Self::Error> {
```

Comment:
The WebSocket would define a custom additional data that would be sent along with the ping, and to be received back with the `pong`. (`Socketto` limits this to 125 bytes).
It may be useful for cases where the `pong` data is different from what we provided in the `ping` request, but I believe those cases to be improbable. Another use-case might arise from submitting multiple pings, where we submit `ping-1 ping-2 ping-3` and the server can reply (as per what I understand from the spec) directly to a newer message (ie - `ping-2` or `ping-3`). 
But implementing such logic seems overkill for our needs.

As we don't send any data, I've removed entirely the `data` section 😄